universal-login-sdk/test/helpers/createWallet.ts

```diff
@@ -0,0 +1,11 @@
+import {Wallet, utils} from 'ethers';
+import {WalletService} from '../../lib/core/services/WalletService';
+
+export const createWallet = async (name: string, walletService: WalletService, wallet: Wallet) => {
```

Comment:
deduplicate?

Reply:
index.ts should not be aware of `test/`, no?